<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>510</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
     <weight>3</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 10pt &quot;Microsoft YaHei&quot;;
border-image: url(:/Resources/images/Button.jpg);
</string>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </widget>
  <widget class="QTableView" name="rootTable">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>200</width>
     <height>450</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="alternatingRowColors">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>510</y>
     <width>200</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(127, 127, 127);</string>
   </property>
   <widget class="QPushButton" name="rootDeleteButton">
    <property name="geometry">
     <rect>
      <x>39</x>
      <y>0</y>
      <width>41</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(158, 187, 243);</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchCata_1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>0</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rootAddButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(158, 187, 243);</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="shortcut">
     <string>Ctrl+R</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>1</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Resources/images/search.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QTableView" name="accountTable">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>200</width>
     <height>450</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>510</y>
     <width>200</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(247, 247, 247);</string>
   </property>
   <widget class="QPushButton" name="accountAddButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>41</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(158, 187, 243);</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="shortcut">
     <string>Ctrl+A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="accountDeleteButton">
    <property name="geometry">
     <rect>
      <x>39</x>
      <y>0</y>
      <width>41</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(158, 187, 243);</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchCata_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>0</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>1</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0)</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Resources/images/search.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>1</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>SizeVerCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::HLine</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>1</width>
     <height>520</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::VLine</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>560</y>
     <width>400</width>
     <height>1</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::HLine</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_4">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>30</y>
     <width>1</width>
     <height>520</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::VLine</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>50</y>
     <width>91</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;MV Boli&quot;;
background:transparent</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Account:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="accountEdit">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>60</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>120</y>
     <width>101</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;MV Boli&quot;;
background:transparent</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Password:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordEdit">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>130</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>250</y>
     <width>41</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;MV Boli&quot;;
background:transparent</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Url:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="urlEdit">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>260</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>400</y>
     <width>101</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MV Boli&quot;;
background:transparent</string>
   </property>
   <property name="text">
    <string> Security：</string>
   </property>
  </widget>
  <widget class="QLabel" name="securityLabel">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>410</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Code Pro</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>510</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 10pt &quot;Microsoft YaHei&quot;;
border-image: url(:/Resources/images/Button.jpg);</string>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setRandomAccount">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>60</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 9pt &quot;Microsoft YaHei&quot;;
border-image: url(:/Resources/images/Button.jpg);</string>
   </property>
   <property name="text">
    <string>random</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setRandomPassword">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>130</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 9pt &quot;Microsoft YaHei&quot;;
border-image: url(:/Resources/images/Button.jpg);</string>
   </property>
   <property name="text">
    <string>random</string>
   </property>
  </widget>
  <widget class="QLabel" name="backgroundLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>550</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Resources/images/background.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="min1Button">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>0</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#min1Button
{
	background:transparent
}
QPushButton#min1Button
{
    border-image: url(:/Resources/images/min.png);
}
QPushButton#min1Button:hover
{
	border-image: url(:/Resources/images/min_hover.bmp);
}
QPushButton#min1Button:pressed
{
    border-image: url(:/Resources/images/min_press.bmp);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="close1Button">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>0</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#close1Button
{
	background:transparent
}
QPushButton#close1Button
{	
    border-image: url(:/Resources/images/close.png);
}
QPushButton#close1Button:hover
{
	border-image: url(:/Resources/images/close_hover.png);
}
QPushButton#close1Button:pressed
{
    border-image: url(:/Resources/images/close_press.png);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>400</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MV Boli&quot;;
background:transparent</string>
   </property>
   <property name="text">
    <string>LittleSecret</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MV Boli&quot;;
background:transparent</string>
   </property>
   <property name="text">
    <string>     Root Catalogue</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MV Boli&quot;;
background:transparent</string>
   </property>
   <property name="text">
    <string>      Sub Catalogue</string>
   </property>
  </widget>
  <zorder>backgroundLabel</zorder>
  <zorder>exitButton</zorder>
  <zorder>rootTable</zorder>
  <zorder>widget</zorder>
  <zorder>accountTable</zorder>
  <zorder>widget_2</zorder>
  <zorder>frame</zorder>
  <zorder>frame_2</zorder>
  <zorder>frame_3</zorder>
  <zorder>frame_4</zorder>
  <zorder>label</zorder>
  <zorder>accountEdit</zorder>
  <zorder>label_3</zorder>
  <zorder>passwordEdit</zorder>
  <zorder>label_4</zorder>
  <zorder>urlEdit</zorder>
  <zorder>label_5</zorder>
  <zorder>securityLabel</zorder>
  <zorder>saveButton</zorder>
  <zorder>setRandomAccount</zorder>
  <zorder>setRandomPassword</zorder>
  <zorder>min1Button</zorder>
  <zorder>close1Button</zorder>
  <zorder>line_3</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>label_9</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
